client/blocks/editor-launch-modal/index.tsx

```diff
@@ -14,7 +14,14 @@ interface Props {
 }
 
 const EditorLaunchModal: React.FunctionComponent< Props > = ( { siteId, locale } ) => {
-	return <FocusedLaunchModal onClose={ noop } siteId={ siteId } locale={ locale } />;
+	return (
+		<FocusedLaunchModal
+			onClose={ noop }
```

Comment:
This line should be deleted, since the `onClose` property has been removed from the `<FocusedLaunchModal />` component